README.md

```diff
@@ -15,51 +15,24 @@ $ vagrant box add --name princeton_box images/ubuntu-16.04.virtualbox.box
 
 ## Prerequisites of MacOS
 
- * `brew install virtualbox`
- * `brew install vagrant`
- * `brew install pyenv`
- * `brew install pyenv-virtualenv`
+ * `brew cask install virtualbox`
+ * `brew cask install vagrant`
+ * `brew install python`
+ * `brew install pipenv`
 
 ## Prerequisites of Ubuntu Bionic
 
  * `sudo add-apt-repository multiverse && sudo apt -y update`
  * `sudo apt -y install virtualbox`
- * `sudo apt-get install vagrant`
- * `git clone https://github.com/pyenv/pyenv.git ~/.pyenv`
```

Comment:
Not a criticism, but is there a reason why pyenv is deprecated entirely?  From the documentation, it seems as if pipenv and pyenv can coexist in an operating system environment: https://pipenv.readthedocs.io/en/latest/#pipenv-features

Reply:
simplicity mostly. We aren't building python libraries in this repo. If one isn't a native python dev having less steps to get to writing your tests is a good thing (IMO). pipenv shell will install ansible and molecule in your virtual environment and you can "get on with it".